database/migration/migrateData.ts

```diff
@@ -723,6 +723,37 @@ const migrateData = async () => {
     `)
   }
 
+  // v0.6.0
+  if (databaseVersionLessThan('0.6.0')) {
+    console.log(' - Update FK reference to trigger_queue in action_queue')
+    await DB.changeSchema(`
+      ALTER TABLE action_queue DROP CONSTRAINT IF EXISTS   
+        action_queue_trigger_event_fkey; 
+      ALTER TABLE action_queue ADD CONSTRAINT action_queue_trigger_event_fkey
+        FOREIGN KEY (trigger_event) REFERENCES trigger_queue (id) ON DELETE CASCADE;
+    `)
+
+    console.log(' - Add application_id column to trigger_queue & action_queue')
+    if (!(await DB.checkColumnExists('trigger_queue', 'application_id'))) {
```

Comment:
Added this additional check (in addition to the version check), because this is quite a slow process, so we want to make sure it only ever runs once.